app/controllers/fhir/r4/api_controller.rb

```diff
@@ -77,12 +77,19 @@ def show
   #
   # PUT /fhir/r4/[:resource_type]/[:id]
   def update
-    status_unsupported_media_type && return unless content_type_header?
+    if request.patch?
```

Comment:
I can't comment on the actual line, but please add patch to the capability statement [here](https://github.com/SaraAlert/SaraAlert/blob/0fb3d499fb09d05c87b743a5daa0adfbc54cf2d6/app/controllers/fhir/r4/api_controller.rb#L391)
`FHIR::CapabilityStatement::Rest::Resource::Interaction.new(code: 'patch')`

Reply:
Added in https://github.com/SaraAlert/SaraAlert/pull/522/commits/e13edb4b4defadbdee354a1fcedfe15110c0ae1d